ajax/libs/venn.js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "venn.js",
+  "filename": "venn.min.js",
+  "description": "A javascript library for laying out area proportional venn and euler diagrams",
+  "homepage": "https://benfred.github.io/venn.js/",
+  "author": "Ben Frederickson (ben@benfrederickson.com)",
+  "keywords": [
+    "venn-diagram",
+    "javascript",
+    "d3",
+    "euler-diagram",
+    "venn",
+    "euler"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/benfred/venn.js.git"
+  },
+  "license": "MIT",
+  "npmName": "venn.js",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/!(bundle.*|*.zip)"
+      ]
+    },
+    {
+      "basePath": "",
+      "files": [
+        "venn*"
```

Comment:
The file `venn.js` exists both under the root directory and `build`. Since this rule will be execute later than the last one rule, `venn.js` under root directory will overwrite the one under `build`. A problem come out: what if these two files are not the same.
You can find these problem in v0.2.9, this is the checksum (sha1sum) of them:
```
885475ad6983fc939b95191e1cafa0b6e18ef213  0.2.9/build/venn.js
8ee6c03781fc892fced4072a729a573cf7fc4478  0.2.9/venn.js
```

Might need to open a issue to ask about this.

Reply:
```
$ diff venn.js build/venn.js
921,949d920
<     /// Uses multidimensional scaling to approximate a first layout here
<     function classicMDSLayout(areas) {
<         // bidirectionally map sets to a rowid  (so we can create a matrix)
<         var sets = [], setids = {};
<         for (var i = 0; i < areas.length; ++i ) {
<             var area = areas[i];
<             if (area.sets.length == 1) {
<                 setids[area.sets[0]] = sets.length;
<                 sets.push(area);
<             }
<         }
<
<         // get the distance matrix, and use to position sets
<         var distances = getDistanceMatrices(areas, sets, setids).distances;
<         var positions = mds.classic(distances);
<
<         // translate rows back to (x,y,radius) coordinates
<         var circles = {};
<         for (i = 0; i < sets.length; ++i) {
<             var set = sets[i];
<             circles[set.sets[0]] = {
<                 x: positions[i][0],
<                 y: positions[i][1],
<                 radius:  Math.sqrt(set.size / Math.PI)
<             };
<         }
<         return circles;
<     }
<
1218a1190,1191
>     /*global d3 console:true*/
>
1745c1718
<     var version = "0.2.8";
---
>     var version = "0.2.9";
1759d1731
<     exports.classicMDSLayout = classicMDSLayout;
1775c1747
< }));
---
> }));
\ No newline at end of file
```

Reply:
it doesn't look like this older version is going to be fixed.

will this package not be pushed without the fix to the older `v0.2.9`?

Reply:
@stephengroat I replied there, let's see the reply 🙂